.github/workflows/update-toast.yml

```diff
@@ -0,0 +1,43 @@
+name: Check Litestar Release
+
+on:
+  schedule:
+    - cron: '0 0 * * *'  # Run every night at midnight
```

Comment:
i wonder if we can use [repository_dispatch](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#repository_dispatch) to dynamically do this instead of killing trees

Reply:
Hard to argue when you frame it like that :D

Reply:
Will try to from `litestar` repo one day 😬